t/debug/debug-mode.t

```diff
@@ -238,11 +238,9 @@ passed
 GET /hello
 --- yaml_config eval: $::yaml_config
 --- response_headers
-Apisix-Plugins: response-rewrite, limit-conn, limit-count, response-rewrite, response-rewrite
+Apisix-Plugins: response-rewrite, limit-conn, limit-count
 --- response_body
 yes
---- error_log
```

Comment:
We still need to check the `core.log.warn("Apisix-Plugins: ", core.table.concat(t, ", "))`

Reply:
done